happi/cli.py

```diff
@@ -288,6 +288,28 @@ def add(ctx, clone: str):
         logger.info('Aborting')
 
 
+@happi_cli.command()
+@click.argument('name', type=str)
+@click.pass_context
+def delete(ctx, name: str):
+    """
+    Delete an existing entry.  Only accepts exact names
+    """
+    client = ctx.obj
+    try:
+        item = client.find_item(name=name)
+    except SearchError as e:
+        raise click.ClickException(f'Could not find item ({name}): {e}')
+
+    item.show_info()
+    if click.confirm('Are you sure you want to delete this entry?'):
```

Comment:
Small suggestion: it might be appropriate to let the user know that they can mark items as inactive instead of deleting them. I know sometimes you absolutely want to nuke an entry but a lot of the time I think people will stumble upon the delete command and not know you can do:

```
happi edit my_device_name active=false
```

Reply:
added in [660ed1c](https://github.com/pcdshub/happi/pull/294/commits/660ed1cc4e85227617b635628a9a0edc061d66dd)